src/app/dataset-view/additional-components/settings-component/settings.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="custom-container px-4 p-responsive">
+    <div class="Layout-sidebar">
+        <div class="pr-md-4 pr-0">
+            <div>
+                <nav aria-label="User settings" data-view-component="true" class="NavigationList">
+                    <ul
+                        role="list"
+                        aria-label="User settings"
+                        nav_classes="org-sub-menu"
+                        data-view-component="true"
+                        class="ActionList"
+                    >
+                        <li
+                            data-item-id=""
+                            data-view-component="true"
+                            [ngClass]="{
+                                'ActionList-item--navActive': true
```

Comment:
Same here, please don't use camelCase in class names.
```suggestion
                                'action-list-item--nav-active': true
```

Same for the rest of the page, and styles

Reply:
Fixed